components/script/dom/cssstyledeclaration.rs

```diff
@@ -342,9 +345,44 @@ impl CSSStyleDeclaration {
     }
 }
 
+lazy_static! {
+    static ref ENABLED_LONGHAND_PROPERTIES: Vec<LonghandId> = {
+        // The 'all' shorthand contains all the enabled longhands, with 2 exceptions:
+        // 'direction' and 'unicode-bidi'
+        let mut v: Vec<LonghandId> = ShorthandId::All.longhands().collect();
```

Comment:
```suggestion
        let mut enabled_properties: Vec<LonghandId> = ShorthandId::All.longhands().collect();
```

Reply:
Used `enabled_longhands` because this doesn't include shorthands.